eng/kubernetes/development/data-management-service-deployment.yaml

```diff
@@ -18,7 +18,6 @@ spec:
       containers:
         - name: data-management-service
           image: local/edfi-data-management-service:latest
-          imagePullPolicy: Never
```

Comment:
Andony seemed to think this line was necessary. Are you sure we can delete it?

Reply:
It was my bad, I removed it when I was testing the image pull from docker hub.